### DISCUSSION TITLE
Is it possible to get notified when a new version is released ?

### DISCUSSION THREAD

**Discussion by rp1231** [LSPosed/MagiskOnWSALocal#523] [Q&A]
I use the "slow" releases and check on https://store.rg-adguard.net/ for new releases by entering the wsa product id.
But the url doesn't show any different path when I enter the search query.

Is there any way to be notified when there is a new release?

Thanks

**Comment by fromhgbwithluv** [comment]
Yes, you can but need some code on Github Action. You can take a look in my Github repository
https://github.com/YT-Advanced/WSA-Script/

**Reply by rp1231** [reply]
Thanks, I'll check it out.

**Comment by Almighty188** [comment]
I use https://store.rg-adguard.net/  as well but just like https://github.com/YT-Advanced/WSA-Script/ which is the same version as what MagiskOnWSALocal will build on.  it's still not the latest version which is 2304.40000.3.0 which has been out for atleast 6 days.  https://github.com/microsoft/WSA/discussions will show you what the latest announced version is but until it makes it to https://store.rg-adguard.net/ - it will still not build on the newest announced version.
